src/components/molecules/search/Search.scss

```diff
@@ -14,6 +14,12 @@
 	.dictionary-search-input {
 		display: inline-block;
 		width: 65%;
+
+		@media screen and (max-width: 614px) {
+		
```

Comment:
Tighten this up and/or run formatter. (option + shift + F)

Reply:
Ok